src/components/comment/Comment.scss

```diff
@@ -0,0 +1,3 @@
+.bdl-ThreadedComment-author {
+    font-weight: bold;
+}
```

Comment:
This class is duplicated in `ActionDropdown.scss`. Do we need to do that?

Reply:
Yup, that's a mistake :)